src/pages/Modules.jsx

```diff
@@ -86,11 +133,15 @@ function Modules({ profile }) {
           <input type="checkbox" id="mentor" name="mentor" checked={mentor} onChange={(e) => setMentor(e.target.checked)} />
           Service Area:
           <input type="text" value={serviceArea} onChange={(e) => setServiceArea(e.target.value)} />
-          {/* Caregiver
-        <input type="checkbox" />
-        Mentor
-        <input type="checkbox" /> */}
+          File:
+          <input type="file" defaultValue="" onChange={handleChange} />
```

Comment:
I would make the onChange function different from the function that pushes to firebase, since I think right now, if you upload a file and then change the file, it will still upload the original. I think the pushing to firebase functionality would be better inside the submitForm function